docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang
+
+RUN apt-get update
+RUN apt-get install -y bzip2
+
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh && \
+    /bin/bash ~/miniconda.sh -b -p /opt/conda && \
+    rm ~/miniconda.sh && \
+    ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh && \
+    echo ". /opt/conda/etc/profile.d/conda.sh" >> ~/.bashrc && \
+    echo "conda activate py35" >> ~/.bashrc
+
+RUN /opt/conda/bin/conda create -y -n py35 python=3.5 
+
```

Comment:
It looks to me that if we base the Docker image on python:3.7, it would be easier to install Go compiler instead:

```dockerfile
FROM python:3.7

RUN apt-get update
RUN apt-get install -y curl
RUN curl --silent https://dl.google.com/go/go1.12.5.linux-amd64.tar.gz go.tar.gz | tar -C /usr/local -xzf -
ENV PATH=/usr/local/go/bin:$PATH
```

Reply:
I tried to build the above Docker image. And I can run `pip wheel .` in the container to build the wheel package. However, after installing the package by running `pip install -I pyrecord*.whl`, still I cannot import it:

```
root@5b1d7a99e130:/work/python# pip install -I pyrecordio-0.0.6-cp37-cp37m-linux_x86_64.whl 
Processing ./pyrecordio-0.0.6-cp37-cp37m-linux_x86_64.whl
Installing collected packages: pyrecordio
Successfully installed pyrecordio-0.0.6
You are using pip version 19.0.1, however version 19.1.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
root@5b1d7a99e130:/work/python# python
Python 3.7.2 (default, Feb  6 2019, 12:04:03) 
[GCC 6.3.0 20170516] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pyrecordio
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'pyrecordio'
```

Reply:
> It looks to me that if we base the Docker image on python:3.7, it would be easier to install Go compiler instead:
> 
> ```dockerfile
> FROM python:3.7
> 
> RUN apt-get update
> RUN apt-get install -y curl
> RUN curl --silent https://dl.google.com/go/go1.12.5.linux-amd64.tar.gz go.tar.gz | tar -C /usr/local -xzf -
> ENV PATH=/usr/local/go/bin:$PATH
> ```

It is ok if we want to stick to python 3.7. It is easier to use conda to install and switch python versions.